src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,170 @@
+
+{{site.data.alerts.callout_success}}
+Note that:
+- Clusters with Enterprise Free or Enterprise Trial licenses cannot disable telemetry; if such a cluster signals a telemetry sending error, it will be due to firewall configuration or a network issue.
+- Clusters with no license have a 7-day grace period before needing to install a license. This is useful for ephemeral development clusters.
+- No license key is required for developers running [single-node clusters](#single-node-clusters).
+
+In general, self-hosted clusters without a license key are restricted to development purposes only. Development purposes includes non-production use in an internal development environment for design, prototype, or development purposes only. This restriction is enforced through [license terms and conditions][csl].
+{{site.data.alerts.end}}
+
+## Obtain a license
+
+To obtain a paid **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a>.
+
+To obtain an **Enterprise Free** or **Enterprise Trial** license, take the following steps:
+
+1. Point your web browser to the [CockroachDB {{ site.data.products.cloud }} Console][cloud-console] and [log in]({% link cockroachcloud/authentication.md  %}#console-ui-authentication) as an account with **Organization Admin** permissions.
+1. Select **Organization &raquo; Enterprise Licenses** from the nav bar dropdown. This will bring you to the **Enterprise Licenses** page, which shows a (possibly empty) list of licenses and their keys, with information about the status of each.
+1. Click the **Create License** button. This will bring you to a page called **Get started with CockroachDB Enterprise**.
+1. On this page, you will create an **Enterprise Trial** license or an **Enterprise Free** license.
+    1. To create an **Enterprise Trial** license:
+        1. Fill in the form with the required information.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+    1. To create an **Enterprise Free** license:
+        1. Fill in the form with the required information.
+        1. Toggle the switch called **Find out if my company qualifies for an Enterprise Free license**.
+        1. Fill out the additional information. *By toggling this switch you are legally attesting to the fact that your company revenue meets the requirements of the license*.
```

Comment:
@rmloveland toggling does not legally attest that your company meets the requirements of the license. I think we should remove this line. 

They actually attest at Step E (when they select the T&Cs)

Reply:
removed it!